Description: optee: Re-name libckteec for QTEE
 Re-names libckteec to libckqteec to signify that this is the QTEE specific
 variant of the PKCS#11 library, so that it does not clash with OP-TEE's
 libckteec.
Author: Harshal Dev <harshal.dev@oss.qualcomm.com>
Origin: upstream, https://github.com/qualcomm/minkipc/commit/f9b8329a4b7de1dc8ca0de036f968fa8617a6dff
Last-Update: 2026-10-12
Comment: Backport of the optee-client/CMakeLists.txt part of the upstream
 commit. The optee-test (xtest_qtee) part does not apply, as optee-test is not
 part of this package.
---
Index: pkg-optee-client/CMakeLists.txt
===================================================================
--- pkg-optee-client.orig/CMakeLists.txt
+++ pkg-optee-client/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.10)
-project(ckteec
+project(ckqteec
 	VERSION 0.1.0
 	LANGUAGES C)
 
@@ -64,9 +64,9 @@ set(SRC
 ################################################################################
 # Built library
 ################################################################################
-add_library(ckteec SHARED ${SRC})
+add_library(ckqteec SHARED ${SRC})
 
-set_target_properties(ckteec PROPERTIES
+set_target_properties(ckqteec PROPERTIES
 	VERSION ${PROJECT_VERSION}
 	SOVERSION ${PROJECT_VERSION_MAJOR}
 )
@@ -74,7 +74,7 @@ set_target_properties(ckteec PROPERTIES
 ################################################################################
 # Flags always set
 ################################################################################
-target_compile_definitions(ckteec
+target_compile_definitions(ckqteec
 	PRIVATE -D_GNU_SOURCE
 	PRIVATE -DBINARY_PREFIX="LT"
 )
@@ -86,14 +86,14 @@ target_compile_definitions(ckteec
 ################################################################################
 # Public and private header and library dependencies
 ################################################################################
-target_include_directories(ckteec
+target_include_directories(ckqteec
 	PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/libckteec/include>
 	$<INSTALL_INTERFACE:libckteec/include>
 	PRIVATE libckteec/src
 	PRIVATE include
 )
 
-target_link_libraries(ckteec
+target_link_libraries(ckqteec
 	PRIVATE pthread
 	PRIVATE minkteec
 	PRIVATE minkadaptor
@@ -103,7 +103,7 @@ target_link_libraries(ckteec
 ################################################################################
 # Install targets
 ################################################################################
-install(TARGETS ckteec
+install(TARGETS ckqteec
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
 
